### ISSUE TITLE
[Feature] Persistent settings/toggles via cookies 

### ISSUE THREAD

**Issue by echo6ix** [stashapp/stash#3773]
**Is your feature request related to a problem? Please describe.**
Particular user toggles/settings on scene/images/gallery and settings pages are not remembered with navigating away from a page/refreshing a page.

**Describe the solution you'd like**
Make the following settings persistent using cookies:
- Scenes page
  - Zoom level
  - View
- Images page
  - Zoom level
  - View
- Gallery page
  - Zoom level
  - View
- Settings (tasks) page
  - Expand/collapse card toggles

**Comment by MrX292** [comment]
for Zoom level and View you can set default filters

**Comment by echo6ix** [comment]
> for Zoom level and View you can set default filters

Sure, but each time you want to change the zoom levels persistently you're going to set the default filter? It's a workaround imo, and should just be saved in the cookies.

**Comment by DogmaDragon** [comment]
I agree on zoom levels and toggles, but I diagree with changed the default display mode. Maybe making a setting people can toggle, but the display modes persisting by default would create a lot inconvenience in my opinion.

**Comment by elkorol** [comment]
Also the chevron in the Performers, Studios and Tags page that hides the left pane isn't persistent and would benefit from this
